.github/workflows/format-code.yml

```diff
@@ -67,3 +71,17 @@ jobs:
           git add .
           git commit -m "[CI] Format code"
           git push
+      
+      - name: 'Post reminder to trigger build manually'
+        continue-on-error: true # Never fail the build if this fails
+        if: steps.fork_status.outputs.fork == 'true'
+        uses: actions/github-script@6e5ee1dc1cb3740e5e5e76ad668e3f526edbfe45 # 2.0.0
```

Comment:
The hash isn't super necessary since it's an action maintained by GitHub, but it doesn't hurt to pin it either